en/advanced-topics/jsb-web-difference.md

```diff
@@ -0,0 +1,88 @@
+# The differences between Pure JS and JS-Binding objects
+
+## Properties
```

Comment:
Currently, only this difference?

Reply:
Yes. If we only see the difference of `properties` binding now.
If we find others, I will update this documentation.

Reply:
Ok.